azext_iot/iothub/_help.py

```diff
@@ -309,3 +309,33 @@ def load_iothub_help():
         type: command
         short-summary: Upload a local file as a device to a pre-configured blob storage container.
     """
+
+    helps["iot hub certificate root-authority"] = """
+        type: group
+        short-summary: Manage the certificate root-authority for an IoT Hub instance.
+    """
+
+    helps["iot hub certificate root-authority set"] = """
+        type: command
+        short-summary: Set the certificate root-authority for an IoT Hub instance to a specific version.
+        long-summary: Transition this resource to a certificate on the DigiCert Global G2 root. Before making this
+          transition, please ensure all devices are updated to contain the public portion of the G2 root. Devices
+          will disconnect and reconnect using the new root.
+          We suggest monitoring current connections but another metric may be more appropriate for your situation.
```

Comment:
What is another metric this last sentence is referring to?

Reply:
not sure - these strings are copy paste from the portal.

The portal includes a link that directs to the metrics page but it doesn't auto populate the metrics so "another metric" may just be something the customer defines and monitors themselves.

Reply:
redefined to user defined metric to be less awk